### PR TITLE
Values layout

### PR DESCRIPTION
**Pull request by AndriiManyak**

### CODE REVIEW

File: src/styles/components/_value.scss [hunk 1]
```diff
@@ -85,4 +88,22 @@
...
+      order: 2;
+    }
+
+    &--hackers {
+      order: 3;
+    }
+
+    &--curious {
+      order: 4;
+    }
```
Comment (vadim-ilchenko) [inline src/styles/components/_value.scss]: mb use bootstrap's classes for orders?

File: src/styles/components/_what-we-value.scss [hunk 2]
```diff
@@ -14,11 +14,13 @@
...
   &__wrapper {
     display: grid;
-    grid-template-columns: repeat(3, 435px);
+    grid-template-columns: repeat(3, auto);
     gap: $grid-gutter-width;
   }
 
-  &__value {
-    margin-bottom: 20px;
+  @media (max-width: $xl-breakpoint) {
```
Comment (vadim-ilchenko) [inline src/styles/components/_what-we-value.scss]: you can write it directly in `__wrapper` block
```scss
&__wrapper {
  display: grid;
  @media (max-wirdth: $xl-breakpoint) {
     ... styles
  }
}
```

File: src/styles/components/_value.scss [hunk 3]
```diff
@@ -35,10 +37,11 @@
     display: flex;
     justify-content: space-between;
     opacity: 0;
-
-    transition: opacity 0.3s;
+    transform: translateX(-100%);
```
Comment (vadim-ilchenko) [inline src/styles/components/_value.scss]: avoid such a big values
better to use smth like transform(-100px) to see smooth animation
if you want to hide element completely do the trick with `visibility: hidden` state

https://css-tricks.com/snippets/css/toggle-visibility-when-hiding-elements/

File: src/scripts/_change-content-animation.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ cards.forEach(card => {
   const additional = card.querySelector('.value__additional');
   const main = card.querySelector('.value__main');
 
-  card.addEventListener('mouseenter', (event) => {
+  card.addEventListener('mouseenter', () => {
```
Comment (vadim-ilchenko) [inline src/scripts/_change-content-animation.js]: is it possible to use just `hover` state in css?

**Review by IBelet — APPROVED** [review]

**Pull request merged**